frontend/src/components/dashboard/subdomain/ConfirmationForm.module.scss

```diff
@@ -40,21 +40,34 @@
     justify-content: space-between;
     padding-top: $spacing-xl;
 
+    button[type="submit"] {
```

Comment:
Praise: nice job matching on a required attribute rather than adding a new class, so that we'll easily notice if we inadvertently remove this attribute.